src/main/resources/js/condec.decision.table.js

```diff
@@ -3,34 +3,39 @@
 	let ConDecDecisionTable = function ConDecDecisionTable() {
 	};
 
-	const decisionTableID = "decisionTable-container";
-	const auiTableID = "tbldecisionTable";
 	const dropDownID = "selectDecisionProblem";
-	const alternativeClmTitle = "Solution options (Alternatives and Decision)";
 	let issues = [];
 	let decisionTableData = [];
 	let currentIssue;
 	
 	ConDecDecisionTable.prototype.initView = function () {		
-		this.loadDecisionProblems();
 		document.getElementById("link-distance-input-decision-table").addEventListener("change", function (event){
 			conDecDecisionTable.loadDecisionProblems();
 		});
 		addOnClickEventToDecisionTableButtons();
 		
 		conDecDecisionGuidance.addOnClickListenerForRecommendations();
 		
-		// TODO Add to observable and add updateView method
 		// Register/subscribe this view as an observer
-		//conDecObservable.subscribe(this);
+		conDecObservable.subscribe(this);
 		
+		// Fill view
+		this.updateView();
 	};
 	
-	function addOnClickEventToDecisionTableButtons () {
-		document.getElementById("btnAddCriterion").addEventListener("click", function (event) {
-			conDecDecisionTable.showAddCriteriaToDecisionTableDialog();
-		});
-		document.getElementById("btnAddAlternative").addEventListener("click", function (event) {
+	ConDecDecisionTable.prototype.updateView = function () {
+		this.loadDecisionProblems();
```

Comment:
![image](https://user-images.githubusercontent.com/33954695/93568380-5ed75a80-f990-11ea-8eca-53624af855bc.png)

You could just reload the table data and without reloading the dropdownmenu

Reply:
I implemented updateView like this:
`ConDecDecisionTable.prototype.updateView = function () {
		this.loadDecisionProblems();
	};`